### PR TITLE
refactor(rxInfoPanel): components/rxInfoPanel to elements/Info Panel

### PR DESCRIPTION
**Pull request by manojdas2212**
JIRA: https://jira.rax.io/browse/EO-1093

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM
- [x] QE LGTM

Before
---------
Components > rxInfoPanel
---------

screen shot 2016-08-01 at 4 44 31 pm

After
---------
Elements > Info Panel
---------

screen shot 2016-08-01 at 5 33 20 pm

Full Page
---------
Elements > Info Panel
---------
screen shot 2016-08-01 at 5 35 15 pm

When view All Modules
---------
 And filter by "info"
---------

screen shot 2016-08-01 at 5 36 41 pm

 And filter by "panel"
---------
screen shot 2016-08-01 at 5 36 56 pm

 And filter by "rxInfoPanel"
---------

screen shot 2016-08-01 at 5 37 08 pm

API DOCS
---------

screen shot 2016-08-01 at 5 37 22 pm

### CODE REVIEW

**Comment by manojdas2212** [comment]
@100stacks @CITguy  please take a look for review

File: src/elements/InfoPanel/InfoPanel.meta.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "displayName": "Info Panel",
+    "stability": "experimental",
+    "description": "",
+    "api": "directive:rxInfoPanel",
+    "keywords": ["info","panel","rxInfoPanel"]
```
Comment (CITguy) [inline src/elements/InfoPanel/InfoPanel.meta.json]: Please put keywords on separate lines, indented below the key.

```json
  "keywords": [
    "info",
    "panel",
    "rxInfoPanel"
  ]
```

**Comment by CITguy** [comment]
`infopanel.simple.html` needs updated

* Per the [HTML Styleguide](https://github.com/rackerlabs/encore-ui/wiki/HTML-Styleguide#use-two-space-indentations), update contents to adhere to 2 spaces per indent
* Remove comment at the top of the file.

File: src/elements/InfoPanel/InfoPanel.meta.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "displayName": "Info Panel",
+    "stability": "experimental",
```
Comment (CITguy) [inline src/elements/InfoPanel/InfoPanel.meta.json]: Per acceptance criteria, this value should be `"stable"`. Please update.

File: src/elements/InfoPanel/InfoPanel.meta.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+    "description": "",
```
Comment (CITguy) [inline src/elements/InfoPanel/InfoPanel.meta.json]: Please update value to `This is a generic info panel for use at the top of pages.`

**Comment by manojdas2212** [comment]
@CITguy review comments addressed please take a look

**Comment by glynnis** [comment]
Design LGTM

**Comment by 100stacks** [comment]
LGTM

**Comment by 100stacks** [comment]
@manojdas2212, please squash to 1 commit.

**Comment by manojdas2212** [comment]
@100stacks @CITguy squash done please take a look

File: src/elements/InfoPanel/docs/examples/infopanel.simple.html [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<rx-info-panel panel-title="A Custom Title">
+   You can put whatever content you want to inside here
+</rx-info-panel>
+
```
Comment (CITguy) [inline src/elements/InfoPanel/docs/examples/infopanel.simple.html]: Please remove blank line at end of file.

**Comment by CITguy** [comment]
I'm going to mark QE LGTM, considering that no tests have changed, and CI services are passing.

**Comment by CITguy** [comment]
Dev LGTM

**Pull request merged**